sensing/autoware_pointcloud_preprocessor/src/concatenate_data/concatenate_and_time_sync_nodelet.cpp

```diff
@@ -94,12 +94,18 @@ PointCloudConcatenateDataSynchronizerComponent::PointCloudConcatenateDataSynchro
     }
     declare_parameter("input_topics", std::vector<std::string>());
     input_topics_ = get_parameter("input_topics").as_string_array();
+
+    // Check if input topics are empty
     if (input_topics_.empty()) {
-      RCLCPP_ERROR(get_logger(), "Need a 'input_topics' parameter to be set before continuing!");
+      RCLCPP_ERROR(get_logger(), "Need 'input_topics' parameter to be set before continuing!");
```

Comment:
Now:
```
"Need 'input_topics' parameter to be set before continuing!"
```

My proposal:
```
"Missing 'input_topics' parameter. Skipping PointCloudConcatenateDataSynchronizerComponent initialization."
```

Honestly I'm not understanding the behavior of the ```get_logger()```. If this logger outputs the component name as default, then my proposal becomes simply ```"Missing 'input_topics' parameter"```